src/lib/layer-manager.js

```diff
@@ -198,6 +207,22 @@ export default class LayerManager {
 
   // PRIVATE METHODS
 
+  _getPickingBuffer() {
+    const {gl} = this.context;
+
+    // Set up a frame buffer if needed
+    if (this.context.pickingFBO === null ||
+      gl.canvas.width !== this.context.pickingFBO.width ||
+      gl.canvas.height !== this.context.pickingFBO.height) {
+      this.context.pickingFBO = new FramebufferObject(gl, {
```

Comment:
As @ibgreen mentioned, let take the chance and update deck.gl with the new naming - Framebuffer